Core/Src/analyzer.c

```diff
@@ -351,9 +345,21 @@ void calc_pack_voltage_stats(acc_data_t *bmsdata)
 		}
 	}
 
+	float real_total_volt = 0;
+	for (int chip = 0; chip < NUM_CHIPS; chip++) {
+		for (int i = 0; i < NUM_CELLS_PER_CHIP; i++) {
+			real_total_volt +=
+				getVoltage(bmsdata->chip_data[chip].voltage[i]);
```

Comment:
see my questions in slack about chip_data.  We shouldnt call getVoltage on a struct field outside of adbms, that feels wierd and takes their specific register format to the rest of the code.  Imo we should either remove chipdata_t and call stuff on cell_asic or store the cell_asic stuff as floats post-conversion.